### PR TITLE
AC-815 repo setup

### PR DESCRIPTION
**Pull request by fredmaggiowski**
Repo setup includes:

- [x] Test analysis and fixes after all the edits
- [X] License updates
- [X] GitHub pages manual deploy preparation
- [X] NPM manual publish preparation
- [X] Travis CI setup
- [X] Coveralls setup

### CODE REVIEW

File: packages/api-explorer/__tests__/Response.test.jsx [hunk 1]
```diff
@@ -74,5 +74,6 @@ test('should show different component tabs based on state', () => {
   expect(doc.html().includes('Response Headers')).toBe(true);
```
Comment (dibericky) [inline packages/api-explorer/__tests__/Response.test.jsx]: We could stop Response Test with the check of ResponseBody prop and ResponseMetadata.
In ResponseMetadata's test we check RequestData, ResponseHeaders, ecc...

File: packages/api-explorer/__tests__/lib/oas-to-har.test.js [hunk 2]
```diff
@@ -43,19 +43,19 @@ describe('url', () => {
...
+      [extensions.PROXY_ENABLED]: false,
     });
     test('should not be prefixed with without option', () => {
       expect(oasToHar(proxyOas, { path: '/path', method: 'get' }).log.entries[0].request.url).toBe(
         'https://example.com/path',
       );
     });
 
-    test('should be prefixed with try.readme.io with option', () => {
+    test('should never be prefixed with try.readme.io with option', () => {
```
Comment (dibericky) [inline packages/api-explorer/__tests__/lib/oas-to-har.test.js]: Should we do this kind of test?

File: packages/api-explorer/src/AuthBox.jsx [hunk 3]
```diff
@@ -117,7 +117,7 @@ class AuthBox extends Component {
 }
 
 AuthBox.propTypes = {
-  securityTypes: PropTypes.object.isRequired,
+  securityTypes: PropTypes.shape({}).isRequired,
```
Comment (dibericky) [inline packages/api-explorer/src/AuthBox.jsx]: We should set the shape of the object

Comment (dibericky) [reply packages/api-explorer/src/AuthBox.jsx]:
```
const oauth2Types = PropTypes.shape({
  OAuth2: PropTypes.arrayOf(PropTypes.shape({
    _key: PropTypes.string,
    type: PropTypes.string
  }))
})

const basicTypes = PropTypes.shape({
  Basic: PropTypes.arrayOf(PropTypes.shape({
    _key: PropTypes.string,
    scheme: PropTypes.string,
    type: PropTypes.string
  }))
})

const apikeyTypes = PropTypes.shape({
  Query: PropTypes.arrayOf(PropTypes.shape({
    _key: PropTypes.string,
    in: PropTypes.string,
    name: PropTypes.string,
    type: PropTypes.string
  }))
})

secutityTypes: PropTypes.shape(
  PropTypes.oneOfTypes([
    oauth2Types,
    basicTypes,
    apikeyTypes
  ])
)
```

File: packages/api-explorer/src/components/ContentWithTitle/index.jsx [hunk 4]
```diff
@@ -1,3 +1,6 @@
+/* eslint-disable no-unused-vars */
```
Comment (dibericky) [inline packages/api-explorer/src/components/ContentWithTitle/index.jsx]: ?

File: packages/api-explorer/__tests__/Doc.test.jsx [hunk 5]
```diff
@@ -185,7 +184,7 @@ describe('onSubmit', () => {
       });
   });
 
-  test('should make request to the proxy url if necessary', () => {
+  test('should not make request to the proxy url if necessary (proxy feature has been removed)', () => {
```
[the remark this answers is not in the document]
Comment (fredmaggiowski) [reply packages/api-explorer/__tests__/Doc.test.jsx]: Let's keep it, if we decide in the future to supporto the Proxy extension we already have tests.

File: packages/api-explorer/__tests__/Response.test.jsx (hunk 1 above)
Comment (fredmaggiowski) [reply packages/api-explorer/__tests__/Response.test.jsx]: I'd not change the original tests if not strictly required

File: packages/api-explorer/__tests__/lib/oas-to-har.test.js (hunk 2 above)
Comment (fredmaggiowski) [reply packages/api-explorer/__tests__/lib/oas-to-har.test.js]: as before

File: packages/api-explorer/src/AuthBox.jsx (hunk 3 above)
Comment (fredmaggiowski) [reply packages/api-explorer/src/AuthBox.jsx]: Changed accordingly using `oneOfType`.

File: packages/api-explorer/src/components/ContentWithTitle/index.jsx (hunk 4 above)
Comment (fredmaggiowski) [reply packages/api-explorer/src/components/ContentWithTitle/index.jsx]: removed unused property

**Pull request merged**